### PR TITLE
Fix display disappearance, add new field, fix ctrl-c

### PR DESCRIPTION
**Pull request by harshil21**
Cleanup after #89

### CODE REVIEW

File: airbrakes/hardware/imu.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def stop(self) -> None:
             self.get_imu_data_packets()
 
         with contextlib.suppress(multiprocessing.TimeoutError):
-            self._data_fetch_process.join(timeout=1)
+            self._data_fetch_process.join(timeout=2)
```
Comment (JacksonElia) [inline airbrakes/hardware/imu.py:92]: no magic numbers >:(

File: airbrakes/hardware/imu.py [hunk 2]
```diff
@@ -99,7 +99,7 @@ def get_imu_data_packet(self) -> IMUDataPacket | None:
         :return: an IMUDataPacket object containing the latest data from the IMU. If a value is not
             available, it will be None.
         """
-        return self._data_queue.get(timeout=1)
+        return self._data_queue.get(timeout=3)
```
Comment (JacksonElia) [inline airbrakes/hardware/imu.py:102]: >:(

File: airbrakes/mock/display.py [hunk 3]
```diff
@@ -128,23 +130,23 @@ def update_display(self) -> None:
...
-            # Don't print the flight data if we are in debug mode
-            if not self._args.debug:
-                self._update_display(False)
-            else:
-                break
-
-    def _update_display(self, end_sim: Literal["natural", "interrupted"] | bool = False) -> None:
+            self._update_display(False)
+
+        if self.end_sim_natural.is_set():
```
Comment (JacksonElia) [inline airbrakes/mock/display.py:140]: comment this, idk whats happening

File: airbrakes/mock/display.py [hunk 4]
```diff
@@ -128,23 +130,23 @@ def update_display(self) -> None:
...
-                self._update_display(self.STATE_END)
-                break
-            # Don't print the flight data if we are in debug mode
-            if not self._args.debug:
-                self._update_display(False)
-            else:
-                break
-
-    def _update_display(self, end_sim: Literal["natural", "interrupted"] | bool = False) -> None:
+            self._update_display(False)
```
Comment (JacksonElia) [inline airbrakes/mock/display.py:138]:
```suggestion
            self._update_display()
```
It is a default arg

File: airbrakes/mock/display.py [hunk 5]
```diff
@@ -128,23 +130,23 @@ def update_display(self) -> None:
...
+
+        if self.end_sim_natural.is_set():
+            self._update_display(self.NATURAL_END)
+        if self.end_sim_interrupted.is_set():
+            self._update_display(self.INTERRUPTED_END)
+        if not self._args.mock and self._airbrakes.state.name == "MotorBurnState":
+            self._update_display(self.STATE_END)
+
+    def _update_display(
```
Comment (JacksonElia) [inline airbrakes/mock/display.py:147]: if we are passing a string literal here, it seems weird to also have the option to pass in a boolean, especially if we are never really setting this boolean to true.

Also idk how much I like string literals. Would maybe prefer enums.

**Review by JacksonElia — CHANGES_REQUESTED** [review]
Looks good, just some nits

File: airbrakes/mock/display.py (hunk 3 above)
Comment (JacksonElia) [reply airbrakes/mock/display.py:140]: tbh var name/method names could be better

Comment (harshil21) [reply airbrakes/mock/display.py:140]: suggestions? `class DisplayEndings(Enum)`?

File: airbrakes/mock/display.py (hunk 5 above)
Comment (harshil21) [reply airbrakes/mock/display.py:147]: Yeah I'm changing this to use constants like `1`, `2`, etc.

.
.
.

Hahaha jk. Enums it is.

File: airbrakes/mock/display.py (hunk 4 above)
Comment (harshil21) [reply airbrakes/mock/display.py:138]: The entire display needs to be scrapped and rewritten

File: airbrakes/hardware/imu.py (hunk 1 above)
Comment (harshil21) [reply airbrakes/hardware/imu.py:92]: but I like magic :(

File: airbrakes/mock/display.py (hunk 3 above)
Comment (JacksonElia) [reply airbrakes/mock/display.py:140]: yeah that sounds good to me

File: airbrakes/mock/display.py (hunk 4 above)
Comment (JacksonElia) [reply airbrakes/mock/display.py:138]: 💀

**Comment by harshil21** [comment]
Not going to write tests for the display even though it's desperately needed, because #54 is underway

**Review by JacksonElia — APPROVED** [review]

**Pull request merged**